### PR TITLE
Feature/iotcore 238 test

### PR DESCRIPTION
**Pull request by jcalderin**
"NOT MERGE YET"

### CODE REVIEW

**Comment by iariasleon** [comment]
As I commented before in #757 , only add license in these files .feature.

LGTM tests for iotcore-238 functionality

File: test/acceptance/integration/ISSUE-713 CPr-propagate Fiware-Service header in forwarded requests/issue-713.feature [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# Created by Jon at 28/01/2015
+Feature: When the ContextBroker fowards  a requests to a Context Provider, the header Fiware-Service, is fowareded too
+  # Enter feature description here
+#TODO: Check if there is more operations forwarded
```
Comment (fgalan) [inline test/acceptance/integration/ISSUE-713 CPr-propagate Fiware-Service header in forwarded requests/issue-713.feature]: Is this a leftover?

Comment (jcalderin) [inline test/acceptance/integration/ISSUE-713 CPr-propagate Fiware-Service header in forwarded requests/issue-713.feature]: Its a remember

NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**